src/sink.rs

```diff
@@ -99,6 +100,18 @@ impl Sink {
         self.pause.load(Ordering::SeqCst)
     }
 
+    /// Stops playback.  To resume, a new sound must be added with `append()`
+    pub fn stop(&self) {
+        self.stopped.store(true, Ordering::Relaxed);
+    }
+
+    /// Gets whether a sound is stopped.
+    ///
+    /// A stopped sound cannot be resumed, a new one must be started with `append()`.
+    pub fn is_stopped(&self) -> bool {
```

Comment:
I don't think that's a good idea. If I add a 2-seconds sound to a sink then call `is_stopped` 3 seconds later, I'd expect it to return true while it will return false.